examples/getting_started/Transformer_handler_generalized.py

```diff
@@ -0,0 +1,538 @@
+import ast
```

Comment:
is this copy pasted from the examples transformer handler? The duplication will hurt us would recommend having this in 1 place